sdcm/tester.py

```diff
@@ -701,6 +701,25 @@ def get_cluster_aws(self, loader_info, db_info, monitor_info):
             else:
                 db_info['device_mappings'] = []
 
+        additional_ebs_volumes_num = self.params.get("aws_ebs_volume_num")
+        if additional_ebs_volumes_num > 0:
+            disk_char = "fghijklmnop"
+            for i in range(additional_ebs_volumes_num):
```

Comment:
I can suggest to change this loop to something like:

```python
for disk_char in "fghijklmnop"[:additional_ebs_volumes_num]:
    ...
```

Reply:
done